Lib/ntpath.py

```diff
@@ -122,54 +122,169 @@ def join(path, *paths):
 # colon) and the path specification.
 # It is always true that drivespec + pathspec == p
 def splitdrive(p):
-    """Split a pathname into drive/UNC sharepoint and relative path specifiers.
-    Returns a 2-tuple (drive_or_unc, path); either part may be empty.
-
-    If you assign
-        result = splitdrive(p)
-    It is always true that:
-        result[0] + result[1] == p
-
-    If the path contained a drive letter, drive_or_unc will contain everything
-    up to and including the colon.  e.g. splitdrive("c:/dir") returns ("c:", "/dir")
-
-    If the path contained a UNC path, the drive_or_unc will contain the host name
-    and share up to but not including the fourth directory separator character.
-    e.g. splitdrive("//host/computer/dir") returns ("//host/computer", "/dir")
-
-    Paths cannot contain both a drive letter and a UNC path.
-
+    """Split path p conservatively into a drive and remaining path.
+    Returns a 2-tuple, (drive, rest). Either component may be empty.
+
+    If the source path contains a DOS drive (i.e. a letter plus a colon), the
+    remaining path is everything after the colon.
+
+    DOS drive examples:
+
+        splitdrive('C:') == ('C:', '')
+        splitdrive('C:dir') == ('C:', 'dir')
+        splitdrive('C:/') == ('C:', '/')
+        splitdrive('C:/dir') == ('C:', '/dir')
+
+    A UNC path is parsed as follows:
+
+        drive
+        vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv
+        "//" domain "/" junction ["/" junction] ["/" object]
+                                                ^^^^^^^^^^^^
+                                                rest
+
+    The UNC root must be exactly two separators. Other separators may be
+    repeated.
+
+    This is a generalization of the UNC specification in [MS-DTYP] 2.2.57. The
+    latter specifies the file namespace, for which the domain is referred to
+    as "host-name" (more generally "server") and the junction as "share-name".
+    The server is commonly a local or remote network name (i.e. NETBIOS name,
+    DNS name, or IP address). It can also be a non-network server provided by
+    a local redirector. The share is a resource provided by the server, such
+    as a file-system directory.
+
+    UNC drive examples in the file namespace:
+
+        splitdrive('//server/share') == ('//server/share', '')
+        splitdrive('//server///share') == ('//server///share', '')
```

Comment:
This example should be changed to match `PathCchSkipRoot()`. For example:

```
>>> buf.value = r'\\server\\\dir'
>>> hr = PathCchSkipRoot(buf, byref(filepath))
>>> print(filepath.value)
\\dir
```

Corresponding `splitdrive()` result:
```
splitdrive('//server///dir') == ('//server/', '//dir')
```
The share component is an empty string. This example can be moved to a section that discusses malformed drives, particularly a section about paths that misleading normalize as functional paths. In this case the path normalizes as if "dir" is a UNC share component.